src/sagemaker/tensorflow/README.rst

```diff
@@ -15,485 +34,130 @@ is a two-step process.
-    #    constructor.
+    estimator = tf.estimator.Estimator(model_fn=my_model_fn, model_dir=args.model_dir)
+    ...
+    train_spec = tf.estimator.TrainSpec(train_input_fn, max_steps=1000)
+    eval_spec = tf.estimator.EvalSpec(eval_input_fn)
+    tf.estimator.train_and_evaluate(mnist_classifier, train_spec, eval_spec)
 
-Example of a complete ``estimator_fn``
-''''''''''''''''''''''''''''''''''''''
+2. Load input data from the input channels. The input channels are defined when ``fit`` is called. For example:
 
 .. code:: python
 
-  def estimator_fn(run_config, hyperparameters):
-      # Defines the features columns that will be the input of the estimator
-      feature_columns = [tf.feature_column.numeric_column(INPUT_TENSOR_NAME, shape=[4])]
-      # Returns the instance of estimator.
-      return tf.estimator.DNNClassifier(feature_columns=feature_columns,
-                                        hidden_units=[10, 20, 10],
-                                        n_classes=3,
-                                        config=run_config)
-
-More details on how to create a ``tensorflow.estimator`` can be find in `Creating Estimators in tf.estimator <https://www.tensorflow.org/extend/estimators>`_.
-
-An example on how to create a TensorFlow training script with an ``estimator_fn`` can find in this `example <https://github.com/awslabs/amazon-sagemaker-examples/tree/master/sagemaker-python-sdk/tensorflow_iris_dnn_classifier_using_estimators>`_.
+    estimator.fit({'train':'s3://my-bucket/my-training-data',
+                  'eval':'s3://my-bucket/my-evaluation-data'})
 
-
-Using a ``Keras`` model instead of a ``model_fn``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-``tf.keras`` is an full implementation inside TensorFlow of the Keras API. To use a ``tf.keras``
-model for training instead of ``model_fn``, you need to write a ``keras_model_fn``. The base skeleton of
-a ``keras_model_fn`` looks like this:
+   In your training script the channels will be stored in environment variables ``SM_CHANNEL_TRAIN`` and
+   ``SM_CHANNEL_EVAL``. You can add them to your argument parsing logic like this:
 
 .. code:: python
 
-  def keras_model_fn(hyperparameters):
-      # Logic to do the following:
-      # 1. Instantiate the Keras model
-      # 2. Compile the Keras model
-      return compiled_model
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--train', type=str, default=os.environ.get('SM_CHANNEL_TRAIN'))
+    parser.add_argument('--eval', type=str, default=os.environ.get('SM_CHANNEL_EVAL'))
 
+3. Export your final model to path stored in environment variable ``SM_MODEL_DIR`` which should always be
+   ``/opt/ml/model``. At end of training SageMaker will upload the model file under ``/opt/ml/model`` to
+   ``output_path``.
 
-Example of a complete ``keras_model_fn``
-''''''''''''''''''''''''''''''''''''''''
 
-.. code:: python
+Calling fit
```

Comment:
I'd rename this to include mention of the `TensorFlow` class, then make a subheader for talking about the constructor arguments and a subheader for talking about calling fit